src/status_im2/subs/general.cljs

```diff
@@ -198,6 +198,22 @@
  (fn [seed]
    (or seed {:step :intro})))
 
+(re-frame/reg-sub
```

Comment:
you shouldn't register subscription with the same name , you already have root subscription with this name, only root subscriptions can "read" db, other subscriptions must be based on other subscriptions not on db, and better to move default values to view, btw only animation-delay 450 is usefull, in others you don't need to set them to nil or false